### DISCUSSION TITLE
What are the actual resolution % we can expect?

### DISCUSSION THREAD

**Discussion by Braydenrud** [merchantprotocol/theadslab#18] [Open Discussion Forum]
I have had a ton of negative experiences where coaches, consultants, etc. over-promise and underdeliver. 

I am very careful in my business to make sure and never put my clients in that situation.

So I need some honest numbers as to what resolution rate people are getting? Does it differ depending on how much traffic the website gets? Are different markets experiencing dramatically different rates?

Looking forward to hearing from you all.

**Comment by jonathonbyrdziak** [comment]
I'll go right to the DB for this one and see what we've done across all websites. Out of the last half a million (500,000) visitors to all websites we were able to identify 197,500. This number includes ALL traffic globally, including repeat traffic to websites.

## That's a total of 39.5% of identities resolved

Out of all of these identities I was only able to match them to 38,887 email addresses in our graph with consumer profile data. Which should certainly not be the case. 

There's an obvious and glaring problem here that we need to resolve. We should be matching more in the arena of 70 to 90% of identities to profiles. I see that you've opened up a [support ticket #16](https://github.com/merchantprotocol/theadslab/issues/16). I will post my findings there and we'll prioritize this.

**Reply by cliffsvr** [reply]
Also if you could describe what do you know about the person at "identity resolved" stage if you don't know their email? Is it just names or device IDs? Some clarity would be great.
